packages/qwik-city/runtime/src/server-functions.ts

```diff
@@ -301,52 +312,57 @@ export const serverQrl: ServerConstructorQRL = (qrl: QRL<(...args: any[]) => any
         const requestEvent = useQwikCityEnv()?.ev ?? this ?? _getContextEvent();
         return qrl.apply(requestEvent, args);
```

Comment:
This is the only place where you should deep freeze the args. This is the code that runs on the server and the args are the ones sent by the client.